dev-games/t4k_common/patches/t4k_common-0.1.1.patchset

```diff
@@ -0,0 +1,86 @@
+From 1252bc2b0bbbb550a5738a6379f0795845ebc73a Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 25 Mar 2017 09:50:43 +0000
+Subject: gcc2 patch
+
+
+diff --git a/configure.ac b/configure.ac
+index 7b54dfb..a7c20c2 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -10,7 +10,7 @@ AC_CANONICAL_HOST
+ AC_CANONICAL_BUILD
+ AC_CANONICAL_TARGET
+ 
+-AM_INIT_AUTOMAKE(foreign)
++AM_INIT_AUTOMAKE([subdir-objects] [foreign])
+ AC_CONFIG_SRCDIR([config.h.in])
+ AC_CONFIG_HEADER([config.h])
+ 
+diff --git a/src/t4k_main.c b/src/t4k_main.c
+index 95d3ec2..97dd368 100644
+--- a/src/t4k_main.c
++++ b/src/t4k_main.c
+@@ -59,7 +59,8 @@ int T4K_HandleStdEvents (const SDL_Event* event)
+   if (event->type != SDL_KEYDOWN)
+     return 0;
+     
+-  SDLKey key = event->key.keysym.sym;
++  {
++  	SDLKey key = event->key.keysym.sym;
+       
+   /* Toggle screen mode: */
+   if (key == SDLK_F10)
+@@ -79,4 +80,5 @@ int T4K_HandleStdEvents (const SDL_Event* event)
+ #endif
+   
+   return ret;
++  }
+ }
+diff --git a/src/t4k_menu.c b/src/t4k_menu.c
+index b1ed842..6b1ab42 100644
+--- a/src/t4k_menu.c
++++ b/src/t4k_menu.c
+@@ -828,7 +828,8 @@ int T4K_RunMenu(int index, bool return_choice, void (*draw_background)(), int (*
+ 
+ 	  first_loop = 0;
+           
+-          int key = event.key.keysym.sym;
++          {
++          	int key = event.key.keysym.sym;
+           
+           if (key == SDLK_F10) {
+ 	    T4K_PrerenderAll();	 // Important when the screen is being RESIZED
+@@ -887,6 +888,7 @@ int T4K_RunMenu(int index, bool return_choice, void (*draw_background)(), int (*
+ 
+           }
+           old_loc = loc;
++          }
+         }
+ 
+         /* handle special action that was caused by an event */
+-- 
+2.7.0
+
+
+From 31d17eb15bc269e470d2b0f189e823fb30fcd015 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 25 Mar 2017 10:36:58 +0000
+Subject: fix missing const for ICONV_CONST
+
+
+diff --git a/src/linebreak/linebreak.c b/src/linebreak/linebreak.c
+index f9c88a1..9cbf4d2 100644
+--- a/src/linebreak/linebreak.c
++++ b/src/linebreak/linebreak.c
+@@ -22,6 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.  */
+  */
+ 
+ #if defined BUILD_MINGW32 && !defined ICONV_CONST
++#elif defined(__HAIKU__) && !defined ICONV_CONST
```

Comment:
Yeah, while I was out today, thought about that option too ... will look into it :)